bricks.js

```diff
@@ -9,3 +9,56 @@ function borderclick(id) {
 	}
 
 } 
+
+function drawpuzzle(width, height) {
+	if (height % 2 == 0) {
+		console.log("Height of brick wall must be odd");
+		return;
+	}
+	var svgns = "http://www.w3.org/2000/svg";
+	for (var y = 0; y < height; y++) {
```

Comment:
`height` sounds like a length to me; I would call this `nRows` or something similar.